src/Storages/System/attachSystemTables.cpp

```diff
@@ -96,105 +96,105 @@ namespace DB
+    attach<StorageSystemBuildOptions>(context, system_database, "build_options", "Contains a list of all build flags, compiler options and commit hash from which this particular server was built.");
+    attach<StorageSystemFormats>(context, system_database, "formats", "Contains a list of all the formats along with flags whether a format is suitable for input/output or whether it supports parallelization.");
+    attach<StorageSystemTableFunctions>(context, system_database, "table_functions", "Contains a list of all available table functions with their descriptions.");
+    attach<StorageSystemAggregateFunctionCombinators>(context, system_database, "aggregate_function_combinators", "Contains a list of all available aggregate function combinator, which could be added to the end of aggregate function and change the way how this aggregate function works.");
+    attach<StorageSystemDataTypeFamilies>(context, system_database, "data_type_families", "Contains a list of all available native data types along with all the aliases which a useful for compatibility with other databases.");
+    attach<StorageSystemCollations>(context, system_database, "collations", "Contains a list of all available collations for alphabetical comparison of strings.");
+    attach<StorageSystemTableEngines>(context, system_database, "table_engines", "Contains a list of all available table engines along with several flags whether a particular table engine supports some feature (e.g. settings, skipping indices, projections, replication, ttl, deduplication, parallel insert etc.)");
+    attach<StorageSystemContributors>(context, system_database, "contributors", "Contains a list of all ClickHouse contributors <3");
+    attach<StorageSystemUsers>(context, system_database, "users", "Contains a list of all users accounts either configured at the server through configuration file or created via SQL.");
+    attach<StorageSystemRoles>(context, system_database, "roles", "Contains a list of all roles created at the server.");
+    attach<StorageSystemGrants>(context, system_database, "grants", "Contains the information about privileges granted to ClickHouse user accounts.");
+    attach<StorageSystemRoleGrants>(context, system_database, "role_grants", "Contains the role grants for users and roles. To add entries to this table, use `GRANT role TO user`. Using this table you may find out which roles are assigned to which users or which roles does user has.");
+    attach<StorageSystemCurrentRoles>(context, system_database, "current_roles", "Contains active roles of a current user. SET ROLE changes the contents of this table.");
+    attach<StorageSystemEnabledRoles>(context, system_database, "enabled_roles", "Contains all active roles at the moment, including current role of the current user and granted roles for current role.");
```

Comment:
Can there be something more that `current role of the current user and granted roles for current role`?

Reply:
I copied it from the documentation.